src/app/app.component.ts

```diff
@@ -14,52 +20,83 @@ import { LoaderService } from './service/loader.service';
 import { HTTP_INTERCEPTORS } from '@angular/common/http';
 import { Observable } from 'rxjs';
 import { CommonModule } from '@angular/common';
+import { MoviesComponent } from './page/movies/movies.component';
 
 @Component({
   selector: 'app-root',
   standalone: true,
   imports: [
     RouterModule,
     RouterOutlet,
-    TagModule,CommonModule,
+    TagModule,
+    CommonModule,
     CarouselModule,
     InputIconModule,
     IconFieldModule,
     ButtonModule,
     InputSwitchModule,
     InputTextModule,
+    MoviesComponent,
     ProgressBarModule,
-    ToolbarModule  ],
-  
+    ToolbarModule,
+  ],
+
   templateUrl: './app.component.html',
   styleUrl: './app.component.scss',
-  changeDetection: ChangeDetectionStrategy.OnPush
+  changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class AppComponent implements OnInit, AfterViewInit {
   title = 'moviesApp';
+  searchTerm: string = '';
+
+  constructor(
+    private route: Router,
+    private service: LoaderService,
+    private cdr: ChangeDetectorRef,
+    private myService: MyServiceService,
+  ) {}
 
-  constructor(private route: Router, private service: LoaderService, private cdr: ChangeDetectorRef) {}
+  test!: MoviesComponent;
 
-  
   ngAfterViewInit(): void {
     // this.cdr.detectChanges()
   }
 
-  isLoading$= this.service.isLoading$;
+  isLoading$ = this.service.isLoading$;
 
   ngOnInit(): void {
-   // this.service.loader.subscribe((res)=>this.isLoading=res)
-
-   this.service.isLoading$.subscribe((res)=>console.log(res))
-
+    // this.service.loader.subscribe((res)=>this.isLoading=res)
   }
 
-  
   navigateHome() {
     this.route.navigate(['/list']);
   }
 
-  navigateMovies(){
-    this.route.navigate(['/movies']);
+  navigateMovies() {
+    this.route.navigate(['movies'], {
+      queryParams: { with_genre: '', startDate: '', endDate: '' },
+    });
+  }
+
+  navigateSeries() {
+    this.route.navigate(['series'], {
+      queryParams: { with_genre: '', startDate: '', endDate: '' },
+    });
   }
+
+  navigateCollection() {
```

Comment:
ეს ოთხი როუტერ ნავიგეიშენ ფუქნცია შეიგიძლია ერთში დაწერო და ჩტმელიდან გადმოაწოდო ლინკი სადაც გინდა რომ გადავიდეს. მაგ: 
navigation(link: string) {
this.route.navigate([link])
}
queryParams შეგიძლია იფით რომელზეც გჭირდება იმას გადააწოდო და თუ არ გჭირდება {} ეს გაუშვი